core/domain/wipeout_jobs_one_off_test.py

```diff
@@ -132,14 +133,22 @@ def test_not_deleted(self):
         self.assertIn(['NOT DELETED', [self.user_1_id]], output)
 
     def test_deleted_success(self):
```

Comment:
Please use better test names here and elsewhere. See guidance in step 2 here: https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests